README.md

```diff
@@ -9,11 +9,28 @@ The `dev-worker` command runs the worker using [wrangler](https://developers.clo
 ```bash
 npm install
 
+# generate a shared secret enabling Reflect Server to authenticate
+# administrative calls, e.g. to create a new room. Configure
+# Reflect Server with the key via wranlger:
+npx wrangler secret put REFLECT_AUTH_API_KEY
```

Comment:
Is it possible/easy to skip the auth check for this? If we're considering having a client-callable version anyway? Cuts down significantly on the mental overhead.

Reply:
the way greg set it up is that the server requires the key be set, otherwise it doesn't make the auth/etc api calls available. so short answer is no, but of course it is software and we can change the server behavior...